stdlib/public/Platform/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+set(swift_platform_name)
+set(swift_platform_flags)
+set(swift_platform_sources
+    Platform.swift
+    Misc.c
+    tgmath.swift.gyb)
+
+if(SWIFT_HOST_VARIANT MATCHES "${SWIFT_DARWIN_VARIANTS}")
+  set(swift_platform_name swiftDarwin)
+  set(swift_platform_sources
+      Darwin.swift
+      ${swift_platform_sources}
+      POSIXError.swift
+      MachError.swift)
+  set(swift_platform_flags
+      SWIFT_COMPILE_FLAGS -Xfrontend -disable-objc-attr-requires-foundation-module
+      API_NOTES_NON_OVERLAY)
+else()
+  set(swift_platform_name swiftGlibc)
+  set(swift_platform_sources
+      Glibc.swift
+      ${swift_platform_sources})
+  set(swift_platform_flags
+      DEPENDS glibc_modulemap)
+endif()
+
+add_swift_library(${swift_platform_name} IS_SDK_OVERLAY
+    ${swift_platform_sources}
+    ${swift_platform_flags})
+
```

Comment:
Removed `INSTALL_IN_COMPONENT stdlib-experimental` because it's [eventually reset to `sdk-overlay`](https://github.com/apple/swift/blob/da60535cedd58c7f6726cb14e821f4473fc214ae/cmake/modules/AddSwift.cmake#L1365).